syntax = "proto3";

package org.seekloud.theia.faceAnalysis.pb;

message ImgData {

  int32 width = 1;
  int32 height = 2;
  int32 pixel_length = 3; //by byte
  bytes data = 4;
}

message ImageReq{
    int64 frame = 1;
    ImgData img = 2;
}

message Point{
    float x=1;
    float y=2;
}

message Mask{
    repeated Point p = 1;
}

message MarkRsp{
    int64 frame = 1;
    repeated Mask l = 2;
}